src/relay/blockchain/events.py

```diff
@@ -11,16 +12,18 @@ def __init__(self, web3_event, current_blocknumber: int, timestamp: int) -> None
         self._web3_event = web3_event
         self.blocknumber: Optional[int] = web3_event.get("blockNumber", None)
         self._current_blocknumber = current_blocknumber
-        # NOTE: The field transactionHash is of type HexBytes sind web3 v4. It can also be a hex string because
-        # the indexer currently can not save bytes in the database.
-        # See issue https://github.com/trustlines-protocol/py-eth-index/issues/16
-        self.transaction_id: hexbytes.HexBytes
-        transaction_id = web3_event.get("transactionHash")
-        if not isinstance(transaction_id, hexbytes.HexBytes):
-            self.transaction_id = hexbytes.HexBytes(web3_event.get("transactionHash"))
-        else:
-            self.transaction_id = transaction_id
+        self._block_hash = _parse_block_hash(web3_event)
+        self.transaction_id = _field_to_hexbytes(web3_event.get("transactionHash"))
         self.type = web3_event.get("event")
+        self._log_index = web3_event.get("logIndex")
+
+    @property
+    def id(self) -> hexbytes.HexBytes:
+        # Create an unique identifier that is resistant to reorgs (block hash)
+        # and multiple events for the same transaction (log index).
```

Comment:
I find the comment confusing, I would just remove it.

Reply:
:heavy_check_mark: